test/e2e/tests/test_layer_version.py

```diff
@@ -30,9 +30,9 @@
 
 RESOURCE_PLURAL = "layerversions"
 
-CREATE_WAIT_AFTER_SECONDS = 10
-UPDATE_WAIT_AFTER_SECONDS = 10
-DELETE_WAIT_AFTER_SECONDS = 10
+CREATE_WAIT_AFTER_SECONDS = 30
+UPDATE_WAIT_AFTER_SECONDS = 30
+DELETE_WAIT_AFTER_SECONDS = 30
```

Comment:
Now that you figured out what was the issue with layer_version tests, shall we reduce those values to 10?

Reply:
Done